lib/routes/devices-sessions.js

```diff
@@ -398,31 +386,32 @@ module.exports = (log, db, config, customs, push, devices) => {
       },
       handler (request, reply) {
         log.begin('Account.deviceDestroy', request)
-
-        const sessionToken = request.auth.credentials
-        const uid = sessionToken.uid
-        const id = request.payload.id
-        let result
-
-        return push.notifyDeviceDisconnected(uid, id)
-          .catch(() => {})
-          .then(() => db.deleteDevice(uid, id))
-          .then(res => {
-            result = res
-            return P.all([
-              request.emitMetricsEvent('device.deleted', {
-                uid: uid,
-                device_id: id
-              }),
-              log.notifyAttachedServices('device:delete', request, {
-                uid: uid,
-                id: id,
-                timestamp: Date.now()
-              })
-            ])
-          })
-          .then(() => result)
-          .then(reply, reply)
+        request.server.methods.devices.cache.drop(request, function() {
```

Comment:
it seems like `.drop` provides an `err` in the callback. Based on https://github.com/hapijs/catbox#api

Should we handle / log the possible error?